### DISCUSSION TITLE
Loading a Stylized Basemap from PMTiles?

### DISCUSSION THREAD

**Discussion by diehl** [opengeos/leafmap#975] [Q&A] [answered]
Instead of using a cloud provider for a basemap, I'm wondering if I can load a basemap via a PMTiles file and style that within Leafmap? If so, I'd appreciate any pointers on how to accomplish this. Thank you!

**Comment by giswqs** [comment, marked as answer]
You can create a map
without a basemap and use m.add_pmtiles() to
load PMTiles

https://geog-312.gishub.org/book/geospatial/maplibre.html#pmtiles

**Reply by diehl** [reply]
Roger that - thanks @giswqs !
